tracer/src/Datadog.Trace/Ci/EventModel/CIVisibilityEventsFactory.cs

```diff
@@ -4,19 +4,23 @@
 // </copyright>
 #nullable enable
 
+using Datadog.Trace.Ci.Tags;
+
 namespace Datadog.Trace.Ci.EventModel;
 
 internal static class CIVisibilityEventsFactory
 {
     public static IEvent FromSpan(Span span)
-        => span.Type switch
+    {
+        return span.Type switch
         {
-            SpanTypes.Test => new TestEvent(span),
+            SpanTypes.Test => span.GetTag(TestTags.Type) == TestTags.TypeBenchmark ? new TestEvent(span, 1) : new TestEvent(span),
```

Comment:
alternative syntatical sugar (just an FYI, no need to change it, don't know that it's better 🤷) 

```suggestion
            SpanTypes.Test when span.GetTag(TestTags.Type) == TestTags.TypeBenchmark => new TestEvent(span, 1),
            SpanTypes.Test => new TestEvent(span),
```

Reply:
I'm changing this right now, wait for the next commit. :D